### ISSUE TITLE
Add https://offtheradar.info

### ISSUE THREAD

**Issue by johenkel** [searxng/searx-instances#318]
### Requirements (make sure to read all of them)

- [X] It is my instance. I bought the domain myself and I own this domain. Free domains (e.g. Freenom) and shared domains (e.g. noip.com) are not allowed.
- [X] I give the right to `check.searx.space` to check my instance (every 3 hours for the response times, every 24 hours for the other tests).
- [X] I acknowledge that managing a public instance is not an easy task and require spending time to keep the instance in good health. E.g. look after your instance by using a monitoring system.
- [X] I guarantee to keep an uptime per month of my instance at **minimum 90%**. Please ask for a removal of your instance if there is a planned long downtime or notify us [here](https://github.com/searxng/searx-instances/discussions/categories/report-instance-downtime) for a short downtime.
- [X] I do not track the users of my instance with any analytics or tracking software.
- [X] I won't try to manipulate the ranking of my instance in a way that give an unfair advantage over the other public instances in the list. (e.g. caching requests for searx.space server)
- [X] I control the final webserver (software) that is serving the requests to the users of my instance. Here is a non-exhaustive list of forbidden hosting types: PaaS, managed (hosting provider controlled) HTTP(S) load balancer (e.g. AWS ALB), Cloudflare, shared Web hosting. TCP load balancer is fine.

### Bot protection

No, I have not installed a bot protection and I do recognize that without one, my instance will be flooded by bots and could have some of its engines stop working (an example is Google).

**Comment by unixfox** [comment]
web server doesn't work closing

**Comment by unixfox** [comment]
We are on a good start for the best uptime here if you don't even have a monitoring system that hasn't alerted you for 2 days... Reopening

**Comment by return42** [comment]
@johenkel please update your instance / thanks!

**Comment by johenkel** [comment]
upgraded to latest.

**Comment by unixfox** [comment]
Make sure you are not blocking our monitoring tool, we can't track the uptime of your instance: https://github.com/searxng/searx-instances-uptime/issues/2463

**Comment by johenkel** [comment]
Whoops sorry about that.
I got regular ufw enabled.  Which ports do I open up ?

Checking here (
https://github.com/searxng/searx-instances-uptime/commits/HEAD/history/offtheradar-info.yml
) it looks like it's been up straight for the past 2 weeks.

**Comment by unixfox** [comment]
There is no ports to open, just whitelist these IP addresses: https://api.github.com/meta or don't block the IP addresses that do requests on the root path (`/`).

I've no idea about your server setup. But we can't keep your instance in the list if our monitoring system can't check its health.

**Comment by johenkel** [comment]
Aye, added IPs from source link to an accept rule.

I got a linode instance running Ubuntu 20.04 with webmin & docker with
searxng, dedicated to just that. Just bare linux firewall rules were
enabled. Nothing out of the ordinary.

I fully understand your rules. I apologize for making extra trouble for you
guys !

**Comment by unixfox** [comment]
Thank you for trying to fix this issue but as you can see there is still a problem your instance is considered as down: https://github.com/searxng/searx-instances-uptime/issues/2463

Feel free to experiment a bit more, the uptime checker is ran every 5 minutes.

**Comment by unixfox** [comment]
Hello @johenkel,

This is a friendly reminder that your provider linode offer IPv6, it would be great if you could add IPv6 to your instance.

It may be not difficult at all, you may already have IPv6 connectivity on your server, all you need is to add a record AAAA in your DNS server.

Here is the documentation here: https://www.linode.com/docs/products/compute/compute-instances/guides/ipv6/

The future of the internet will thank you, see [why here](https://old.reddit.com/r/explainlikeimfive/comments/1k45y7/eli5_what_is_ipv6_and_why_is_it_so_important/).

**Comment by johenkel** [comment]
Thank you,
- upgraded searxng to latest ( 2023.5.22+b2d0b743  )
- enabled ipv6 on my linode instance
- made sure icmp is enabled for the server ( ran : iptables -A INPUT -p
icmp -j ACCEPT )
- made sure linux firewall accepts ipv6
- added ipv6 IP from the linode instance at my dns provider ( namecheap) as
a AAAA record

kept refreshing searx.space to see that it catches my version upgrade. It
has been more than 5min .
when going to offtheradar.info directly it comes all up.

I'll come back tomorrow re-checking everything.

Joerg

**Comment by unixfox** [comment]
IPv6 doesn't seem to be enabled on your domain: https://ready.chair6.net/?url=offtheradar.info

**Comment by return42** [comment]
@johenkel FYI your instance has not been update since a longer time / its recommended to update regularly.

**Comment by johenkel** [comment]
I apologize. Instance has been updated and I will keep up on it better.

**Comment by unixfox** [comment]
Please consider joining our Matrix room for public instance maintainers by joining our Matrix room: https://matrix.to/#/#searxng:matrix.org
then pinging @\unixfox, @\dalf and @\mrpaulblack for asking to be invited to the Matrix room.
We discuss troubles managing a public instance, sharing some advices (like how to protect against bots), announcing big changes in searxng and more.

**Comment by unixfox** [comment]
A new big change has been introduced 🎉!

Please read this documentation for how to proceed: https://gist.github.com/unixfox/4e22730769540fe5b9f1943a86439761

We are going to make mandatory this new parameter by 1st December 2023. Any instance that do not have this parameter enabled won't be allowed in the public list anymore.

We can make exceptions in case you really do not want to enable our bot limiter solution, but you will have to send us proofs: clearly stating how you detect bots, how you actively deal with them and also prove that it really works.

Feel free to ask here if you have any troubles setting up this new parameter.

**Comment by unixfox** [comment]
@johenkel Important step above! The deadline has already passed, if you do not conform we will have to remove your instance by the end of the year.
